Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.13-slim
+FROM python:3.11-slim
```

Comment:
can we revert this to 3.13-slim please, and whilst you're at it pin it to a hash

`FROM python:sha256:193fdd0bbcb3d2ae612bd6cc3548d2f7c78d65b549fcaa8af75624c47474444d # 3.13-slim`

https://hub.docker.com/layers/library/python/3.11-slim/images/sha256-1591aa8c01b5b37ab31dbe5662c5bdcf40c2f1bce4ef1c1fd24802dae3d01052?context=explore